modules/@apostrophecms/schema/ui/apos/components/AposInputRelationship.vue

```diff
@@ -44,17 +47,30 @@ export default {
       }
     }
   },
+  emits: [ 'input' ],
   data () {
     return {
-      browseLabel: 'Browse ' + apos.modules[this.field.withType].pluralLabel,
       searchList: [],
       items: this.value.data || this.listItems,
-      lastSearches: {}
+      lastSearches: {},
+      originalDisabled: this.status.disabled,
+      searching: false
     };
   },
   mounted() {
     this.validateAndEmit();
   },
+  computed: {
+    pluralLabel() {
+      return apos.modules[this.field.withType].pluralLabel;
+    },
+    placeholder() {
+      return this.field.placeholder || `Search ${this.pluralLabel}`;
```

Comment:
I think that if we're going to do this "auto-guess the verbiage" thing we should push it as config from the server side where i18n is easy. I would be open with that being a followup ticket though as long as it gets opened.

Reply:
opened an issue https://apostrophecms.atlassian.net/browse/A30U-436